### PR TITLE
Pure JS action system working in JSB

### PR DESCRIPTION
**Pull request by pandamicro**
Re: cocos-creator/fireball#5420

Changes proposed in this pull request:
 * Move ActionManager to actions module
 * Pure JS action system working in JSB

> Makes sure these boxes are checked before submitting your PR - thank you!
>
- [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
- [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.
- To official teams:
  - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
  - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)
  - [x] Make sure any log information in `cc.log` , `cc.error`, `cc.warn` or `cc.assert` has been moved into `DebugInfos.js` with an ID

@cocos-creator/engine-admins

### CODE REVIEW

File: jsb/jsb-director.js [hunk 1]
```diff
@@ -436,17 +458,11 @@ cc.js.mixin(cc.director, {
     },
 });
 
+cc.defineGetterSetter(cc.director, "actionManager", cc.director.getActionManager, cc.director.setActionManager);
```
Comment (jareguo) [inline jsb/jsb-director.js:461]: 没有这个 API 吧？

File: cocos2d/core/CCNode.js [hunk 2]
```diff
@@ -1362,7 +1361,9 @@ var Node = cc.Class({
      * node.stopAction(action);
      */
     stopAction (action) {
-        cc.director.getActionManager().removeAction(action);
+        if (cc.director._actionManager) {
```
Comment (jareguo) [inline cocos2d/core/CCNode.js:1367]: 什么情况下会没有 cc.director._actionManager ？

File: jsb/jsb-director.js (hunk 1 above)
Comment (pandamicro) [reply jsb/jsb-director.js:461]: jsb 中有

File: cocos2d/core/CCNode.js (hunk 2 above)
Comment (pandamicro) [reply cocos2d/core/CCNode.js:1367]: 不 require actions 模块的情况下

Comment (jareguo) [reply cocos2d/core/CCNode.js:1367]: 那没有 action 的话，是不是这里（stopAction）就应该让他报错，而不是什么也不做？

Comment (pandamicro) [reply cocos2d/core/CCNode.js:1367]: 这里我不是很满意，还没想清楚，我也不想每次用都判断，最好能不要耦合那么强，比如初始 action 相关函数都报 warning，require action 的时候替换函数

**Comment by nantas** [comment]
现在裁剪 action 模块时，能把 actionManager 都干掉了吗？

**Comment by pandamicro** [comment]
> 现在裁剪 action 模块时，能把 actionManager 都干掉了吗？

嗯，可以

**Pull request merged**